.env

```
@@ -1,12 +1,21 @@
-CLOUDINARY_API_KEY=641649343363871
-CLOUDINARY_API_SECRET=n8Nr4bMYGioiE4fssNV2uI6Rcq4
-
-EMAIL_USER=tuemail@gmail.com
-EMAIL_PASS=tu_clave_app_gmail
+PORT=4000
+DATABASE_URL=postgresql://postgres:evCXVApWczdHTSYOoZLdWaUauaMhNFnW@gondola.proxy.rlwy.net:59376/railway
+
+JWT_SECRET=supersecretkey
+
+CLOUDINARY_CLOUD_NAME=dwiga4jg8
+CLOUDINARY_API_KEY=641649343363871
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLOUDINARY_API_KEY key should go before the CLOUDINARY_CLOUD_NAME key

(UnorderedKey)